ios/Mattermost/Info.plist

```diff
@@ -64,25 +64,25 @@
 	<key>NSBluetoothPeripheralUsageDescription</key>
 	<string>Share post data across devices with Mattermost</string>
 	<key>NSCalendarsUsageDescription</key>
-	<string>Share contacts in your Mattermost instance</string>
+	<string>Share contacts in your Mattermost workspace</string>
 	<key>NSCameraUsageDescription</key>
-	<string>Take a Photo or Video and upload it to your Mattermost instance</string>
+	<string>Take a Photo or Video and upload it to your Mattermost workspace</string>
 	<key>NSFaceIDUsageDescription</key>
 	<string>Mattermost uses Face ID to restrict unauthorized users from accessing the app.</string>
 	<key>NSLocationAlwaysUsageDescription</key>
-	<string>Send your current location to your Mattermost instance</string>
+	<string>Send your current location to your Mattermost workspace</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
-	<string>Send your current location to your Mattermost instance</string>
+	<string>Send your current location to your Mattermost workspace</string>
 	<key>NSMicrophoneUsageDescription</key>
-	<string>Capture audio when recording a video to share to your Mattermost instance</string>
+	<string>Capture audio when recording a video to share to your Mattermost workspace</string>
 	<key>NSMotionUsageDescription</key>
-	<string>Share your route in your Mattermost instance</string>
+	<string>Share your route in your Mattermost workspace</string>
 	<key>NSPhotoLibraryAddUsageDescription</key>
-	<string>Save downloaded Photos and Videos from your Mattermost instance to your device</string>
+	<string>Save downloaded Photos and Videos from your Mattermost workspace to your device</string>
```

Comment:
```suggestion
	<string>Save downloaded Photos and Videos from Mattermost to your device</string>
```

Reply:
Question: Is there a specific reason why ``Photo`` and ``Video`` are capitalized here? If not, let's lowercase them, please.